### PR TITLE
more styling options

### PR DESCRIPTION
**Pull request by timschmidtdev**
Screen Shot 2020-12-19 at 3 04 43 PM

### CODE REVIEW

**Comment by timschmidtdev** [comment]
I recommend closing old PRs if you don't merge them like this one #1 
And consider deleting the branch after closing the PR.

**Review by maryschmidtdev — APPROVED** [review]

File: styles.css [hunk 1]
```diff
@@ -16,16 +16,13 @@ h2 {
...
-  width: 20em;
   float: right;
-}
-
-nav a {
-  padding: 10px;
+  justify-content: space-between;
+  margin: auto;
+  width: 22em;
 }
```
Comment (timschmidtdev) [inline styles.css:26]: Some of this stuff in the `nav` didn't work since there was also a `ul` so I moved the styling to the `ul`.

Comment (maryschmidtdev) [reply styles.css:26]: Ok

File: styles.css [hunk 2]
```diff
@@ -80,18 +78,18 @@ a:active {
...
 
 /* Float Clearfix ------------------ */
 
 .group:after {
-  content: "";
-  display: table;
   clear: both;
+  content: '';
+  display: table;
 }
```
Comment (timschmidtdev) [inline styles.css:93]: no changes, just some sorting.
This is the vs code extension I use for sorting sections of code.
Screen Shot 2020-12-19 at 3 09 06 PM

Comment (maryschmidtdev) [reply styles.css:93]: Cool

Comment (maryschmidtdev) [reply styles.css:93]: I don't know what this means - your comment from above. 

"I recommend closing old PRs if you don't merge them like this one #1
And consider deleting the branch after closing the PR."

**Pull request merged**